packages/core/src/resources/Namespaces.ts

```diff
@@ -34,7 +34,11 @@ export interface NamespaceExistsSchema extends Record<string, unknown> {
 
 export class Namespaces<C extends boolean = false> extends BaseResource<C> {
   all<E extends boolean = false, P extends PaginationTypes = 'offset'>(
-    options?: { search?: string; ownedOnly?: string } & PaginationRequestOptions<P> &
+    options?: {
+      search?: string;
+      ownedOnly?: string;
```

Comment:
**question:** I do not know why `ownedOnly` is of type `string` instead of boolean.
I can patch it in another PR

Reply:
Most likely a typo